reference/7.0/Microsoft.PowerShell.Core/About/about_Environment_Variables.md

```diff
@@ -323,6 +327,10 @@ You can add also add a command that changes the value to your profile or use
 System in Control Panel to change the value of the `PSModulePath` environment
 variable in the registry.
 
+When starting Windows PowerShell or PowerShell ISE, PowerShell will remove the
```

Comment:
When starting Windows PowerShell or PowerShell ISE, PowerShell removes the

Remove future tense